tuxemon/technique.py

```diff
@@ -149,9 +149,9 @@ def load(self, slug: str) -> None:
         # technique use notifications (translated!)
         # NOTE: should be `self.use_tech`, but Technique and Item have
         # overlapping checks
-        self.use_item = T.maybe_translate(results.get("use_tech"))
-        self.use_success = T.maybe_translate(results.get("use_success"))
-        self.use_failure = T.maybe_translate(results.get("use_failure"))
+        self.use_item = T.translate(results["use_tech"])
```

Comment:
Is there pydantic validation for these values?

Reply:
How quickly do we find out if they aren't set?

Reply:
You're right. I thought it was necessary for the correct display during battles. But after a series of tests, it wasn't.